apps/image-server/server.js

```diff
@@ -155,26 +148,37 @@ app.get('/image/*',
  *  The url for creating one Image
  */
 app.post('/image',
-  passport.authenticate('bearer', {session: false}),
   upload.single('image'), (req, res, next) => {
     // req.file is the `image` file
     // req.body will hold the text fields, if there were any
     //
-    if (!res.headerSent) {
-      res.setHeader('Content-Type', 'application/json');
+    const createdCombination = secret + req.query.exp_date
```

Comment:
Wat gebeurt er als secret niet gezet is? Is dit dan wel veilig of moet er een foutmelding worden gegeven